Assets/Scripts/UI/PauseMenu.cs

```diff
@@ -100,6 +100,11 @@ public void GoToMainMenu()
         Time.timeScale = 1;
         GameManager.LoadScene(PlayerConstants.BuildSceneIndex);
     }
+    public void ResetLevel()
+    {
+        SceneManager.LoadScene(levelName.text);

```

Comment:
no no no. look at PlayerProgress.cs, there's better way to do there where we dont have to reload the entire scene